overlord/devicestate/firstboot.go

```diff
@@ -42,6 +43,8 @@ var errNothingToDo = errors.New("nothing to do")
 
 var runtimeNumCPU = runtime.NumCPU
 
+var createAllSystemUsers = createAllKnownSystemUsers
```

Comment:
This added name risks creating a bit of confusion. If you just want to make the function mockable, you can change from `func A() {}` to `var A = func() {}`.